zhaquirks/tuya/ts0601_trv_sas.py

```diff
@@ -128,15 +153,15 @@ def map_attribute(self, attribute, value):
 
         if attribute == "occupied_heating_setpoint":
             # centidegree to decidegree
-            return {SASSWEL_HEATING_SETPOINT_ATTR: round(value / 10)}
+            return {SASWELL_HEATING_SETPOINT_ATTR: round(value / 10)}
 
         if attribute == "system_mode":
             # this quirk does not support programmig modes so we force the schedule mode to be always off
             # more details: https://github.com/zigpy/zha-device-handlers/issues/1815
             if value == self.SystemMode.Off:
-                return {SASSWEL_STATE_ATTR: 0, SASSWEL_SCHEDULE_ENABLE_ATTR: 0}
+                return {SASWELL_STATE_ATTR: 0, SASWELL_SCHEDULE_ENABLE_ATTR: 0}
             if value == self.SystemMode.Heat:
-                return {SASSWEL_STATE_ATTR: 1, SASSWEL_SCHEDULE_ENABLE_ATTR: 0}
+                return {SASWELL_STATE_ATTR: 1, SASWELL_SCHEDULE_ENABLE_ATTR: 0}
 
 
 class Thermostat_TYST11_c88teujp(TuyaThermostat):
```

Comment:
This device class is using the same tuya MCU but one older Zigbee module so it shall working the same way
Pleas put the calibration in it so users that having both the new and old version is getting the same functionality.

Reply:
I don't understand. The name used to be like this before.

Reply:
Yes putting in the same as you have doing in old line `195 class Thermostat_TZE200_c88teujp(TuyaThermostat):` so both device classes is getting the same functionality.

Reply:
> I added the calibration to that list. Is that correct?

Not, is not needed.

@MattWestb is asking to you to implement the same code in the `Thermostat_TYST11_c88teujp` class:
https://github.com/zigpy/zha-device-handlers/blob/f66d4335e8f0a95abbc154543054a870bfde0ad9/zhaquirks/tuya/ts0601_trv_sas.py#L167

Just add to it:
* temperature_calibration_bus 
* TuyaTemperatureOffset

You would need to put the new classes upper than the `Thermostat_TYST11_c88teujp` definition

Reply:
Isn't that related to the fix tor the schedule mode that has been implemented lately? I did not do that, but I should include it in the PR whilst were at it?

Reply:
Should be fixed now.